spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/observation/ObservationAutoConfigurationTests.java

```diff
@@ -75,17 +87,25 @@ void autoConfiguresDefaultMeterObservationHandler() {
 			// Observation leads to a timer
 			MeterRegistry meterRegistry = context.getBean(MeterRegistry.class);
 			assertThat(meterRegistry.get("test-observation").timer().count()).isEqualTo(1);
+			assertThat(context).hasSingleBean(DefaultMeterObservationHandler.class);
+			assertThat(context.getBeansOfType(ObservationHandler.class)).hasSize(1);
 		});
 	}
 
+	@Test
+	void allowsDefaultMeterObservationHandlerToBeDisabled() {
+		this.contextRunner.withClassLoader(new FilteredClassLoader(MeterRegistry.class))
+				.run((context) -> assertThat(context).doesNotHaveBean(ObservationHandler.class));
+	}
+
 	@Test
 	void autoConfiguresObservationPredicates() {
 		this.contextRunner.withUserConfiguration(ObservationPredicates.class).run((context) -> {
 			ObservationRegistry observationRegistry = context.getBean(ObservationRegistry.class);
 			// This is allowed by ObservationPredicates.customPredicate
-			Observation.start("observation1", observationRegistry).start().stop();
+			Observation.start("observation1", observationRegistry).stop();
 			// This isn't allowed by ObservationPredicates.customPredicate
-			Observation.start("observation2", observationRegistry).start().stop();
+			Observation.start("observation2", observationRegistry).stop();
```

Comment:
As far as I can tell, this change is unrelated to wrapping the `DefaultMeterObservationHandler` in a `TracingAwareMeterObservationHandler`. If I've got that right, it should be handled separately please.